java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -603,4 +605,58 @@ public void scan_with_options_test() {
                     "Unable to find " + typeKeys.get(type) + " in a scan by match pattern");
         }
     }
+
+    @Test
+    @SneakyThrows
+    public void test_transaction_dump_restore() {
+        GlideString key1 = gs("{key}-1" + UUID.randomUUID());
+        GlideString key2 = gs("{key}-2" + UUID.randomUUID());
+        String value = UUID.randomUUID().toString();
+
+        // Setup
+        assertEquals(OK, client.set(key1, gs(value)).get());
+
+        // Verify dump
+        Transaction transaction1 = new Transaction();
+        transaction1.withBinarySafeOutput();
+        transaction1.dump(key1);
+        Object[] result = client.exec(transaction1).get();
+        GlideString payload = (GlideString) (result[0]);
+
+        // Verify restore
+        Transaction transaction2 = new Transaction();
+        transaction2.withBinarySafeOutput();
```

Comment:
verify this is necessary

Reply:
I need to keep, otherwise it will fail 
assertEquals(gs(value), response[1]);